src/inventory/inventory-menu.jsx

```diff
@@ -2,20 +2,28 @@ import React from "react";
 import { Tabs, Tab } from "carbon-components-react";
 import { InventoryLandingPage } from "./inventory-landing-page";
 import { useCookies } from "react-cookie";
-import { getLocationName, inventoryMenu, locationCookieName } from "../../constants";
-import { DispensePage } from "./dispense/dispense-page"
+import {
+	getLocationName,
+	inventoryMenu,
+	locationCookieName,
+} from "../../constants";
+import { DispensePage } from "./dispense/dispense-page";
+import StockReceipt from "../stock-receipt/stock-receipt";
 
 const InventoryMenu = () => {
 	const [cookies] = useCookies();
 	return (
-		<div style={{ padding: "2rem" }}>
+		<div style={{ paddingTop: "2rem" }}>
 			<h4>{getLocationName(cookies[locationCookieName]?.name)} </h4>
 			<Tabs>
 				<Tab label={inventoryMenu[0]}>
 					<InventoryLandingPage />
 				</Tab>
+				<Tab label="Stock Receipt">
+					<StockReceipt />
+				</Tab>
 				<Tab label={inventoryMenu[1]}>
-					 <DispensePage />
+					<DispensePage />
```

Comment:
I had assumed this PR is only for Stock Receipt screen with eAushadya details. But looks like it also has Dispense Screen UI (with hardcoded data). Plz confirm if this was intentional.

Reply:
this was the basic mock up created by UI devs, which we would continue building screen on top of this.